packages/checkpoint/data_stream/firewall/elasticsearch/ingest_pipeline/default.yml

```
@@ -508,6 +512,10 @@ processors:
         } else if (iana_number == '132') {
             ctx.network.transport = 'sctp';
         }
+  - convert:
+      field: checkpoint.packets
+      type: long
+      ignore_missing: true
   - rename:
       field: checkpoint.packets
       target_field: network.packets
```

Thinking of doing this in a single step - 
```suggestion
  - convert:
      field: checkpoint.packets
      type: long
      target_field: network.packets
      ignore_missing: true
```

---

as above